src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -203,6 +194,35 @@ protected PodSpec createPodSpec(String appId, AppDeploymentRequest request, Inte
 		return podSpec.build();
 	}
 
+	private List<Toleration> getTolerations(AppDeploymentRequest request) {
+		List<Toleration> tolerations = new ArrayList<>();
+
+		String tolerationDeploymentProperty = request.getDeploymentProperties()
+				.getOrDefault("spring.cloud.deployer.kubernetes.tolerations", "");
+		if (!StringUtils.isEmpty(tolerationDeploymentProperty)) {
+			try {
```

Comment:
I did not combine these in case there were changes needed for handling volumes or tolerances in the future if there are changes on how k8s needs them set, but they could definitely be put into a common method as they are now.

Reply:
yes id say go for it

Reply:
I will get this updated as well, thanks, I did not push this change in commit 19721db

Reply:
updated in commit 8056f94